python/py-kivy/files/patch-setup.py.diff

```diff
@@ -18,13 +18,15 @@
          # check the existence of frameworks
          sdl2_valid = True
          sdl2_flags = {
-@@ -1052,9 +1052,6 @@
+@@ -1101,11 +1101,6 @@
              ('Topic :: Software Development :: Libraries :: '
               'Application Frameworks'),
              'Topic :: Software Development :: User Interfaces'],
 -        dependency_links=[
 -            'https://github.com/kivy-garden/garden/archive/master.zip'],
--        install_requires=['Kivy-Garden>=0.1.4', 'docutils', 'pygments'],
-         setup_requires=[
-             'cython>=' + MIN_CYTHON_STRING
-         ] if not skip_cython else [])
+-        install_requires=[
+-            'Kivy-Garden>=0.1.4', 'docutils', 'pygments'
```

Comment:
I feel it better to keep docutils and pygments - just drop Kivy-Garden.

Reply:
done